iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/MultiplexingClientTests.java

```diff
@@ -836,6 +836,7 @@ public void testTwin() throws Exception
 
         for (int i = 0; i < DEVICE_MULTIPLEX_COUNT; i++)
         {
+            testInstance.testDevicesArrayIdentity.get(i).twinUpdated = true;
```

Comment:
the next couple of lines don't make it obvious (to me!) why this property needs to be set, a comment would useful here.